common/base/SharedMemory_win32.cpp

```diff
@@ -27,13 +27,17 @@ SharedMemory::SharedMemory(const std::string& name, size_t size) : mSize(size) {
     const std::string kFileUri = "file://";
     if (name.find(kFileUri, 0) == 0) {
         // Convert URI to path using win32 api.
+#ifdef __MINGW64__
```

Comment:
yeah, think it's the same issue with not building all of the files in `common/base`